app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -554,9 +551,10 @@ class BrowserTabViewModelTest {
     }
 
     @Test
-    fun whenBookmarkEditedThenDaoIsUpdated() = coroutineRule.runBlocking {
-        testee.onSavedSiteEdited(Bookmark(0, "A title", "www.example.com", 0))
-        verify(mockBookmarksDao).update(BookmarkEntity(title = "A title", url = "www.example.com", parentId = 0))
+    fun whenBookmarkEditedThenRepositoryIsUpdated() = coroutineRule.runBlocking {
+        val bookmark = Bookmark(0, "A title", "www.example.com", 0)
```

Comment:
nit: for consistency (and also know what those 0 are) would be nice to add function parameters

```suggestion
        val bookmark = Bookmark(id = 0, title = "A title", url = "www.example.com", parentId = 0)
```